dev/update_datafusion_versions.py

```diff
@@ -44,6 +44,22 @@
     'datafusion-examples': 'datafusion-examples/Cargo.toml',
 }
 
+def update_workspace_version(new_version: str):
```

Comment:
FYI @andygrove  I had to change the `update_datafusion_version.py` script slightly to handle the new workspace version